### PR TITLE
test flux aggregator default oracle gas limits

### PR DESCRIPTION
**Pull request by se3000**

### CODE REVIEW

File: evm-contracts/test/v0.6/FluxAggregator.test.ts [hunk 1]
```diff
@@ -1052,23 +1052,73 @@ describe('FluxAggregator', () => {
...
+
+      // function adverserialQuickselectList(len: number): number[] {
+      //   let xs: number[] = []
+      //   let pi: number[] = []
+      //   for (let i = 0; i < len; i++) {
+      //     pi[i] = i
+      //     xs[i] = 0
+      //   }
+
+      //   for (let l = len - 1; l > 0; l--) {
```
Comment (kaleofduty) [inline evm-contracts/test/v0.6/FluxAggregator.test.ts:1111]:
```suggestion
      //   for (let l = len; l > 0; l--) {
```
There are only two hard things in computer science: cache invalidation, naming things, and off-by-one errors. :-)

Comment (kaleofduty) [reply evm-contracts/test/v0.6/FluxAggregator.test.ts:1111]: I also suggest doing an `assert.deepEqual` against a golden output, e.g. for `len == 16`, you should get `[2, 4, 6, 8, 10, 12, 14, 16, 1, 9, 5, 11, 3, 13, 7, 15]`

File: evm-contracts/src/v0.6/dev/FluxAggregator.sol [hunk 2]
```diff
@@ -81,7 +81,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
    * funds without the owner's intervention.)
    */
   uint256 constant private RESERVE_ROUNDS = 2;
-  uint256 constant private MAX_ORACLE_COUNT = 42;
+  uint256 constant private MAX_ORACLE_COUNT = 77;
```
Comment (samsondav) [inline evm-contracts/src/v0.6/dev/FluxAggregator.sol:84]: Wow we're really going for broke here ;)

File: evm-contracts/test/v0.6/FluxAggregator.test.ts [hunk 3]
```diff
@@ -1052,23 +1052,75 @@ describe('FluxAggregator', () => {
...
+        if (!!tx) {
+          const receipt = await tx.wait()
+          assert.isAbove(400_000, receipt.gasUsed.toNumber())
+        }
+      })
 
-          await aggregator
-            .connect(personas.Carol)
-            .addOracles([fakeAddress], [fakeAddress], minMax, minMax, rrDelay)
+      function adverserialQuickselectList(len: number): number[] {
```
Comment (samsondav) [inline evm-contracts/test/v0.6/FluxAggregator.test.ts:1105]: This is cool

File: evm-contracts/src/v0.6/dev/FluxAggregator.sol (hunk 2 above)
Comment (se3000) [reply evm-contracts/src/v0.6/dev/FluxAggregator.sol:84]: It was the most we could fit in under 400k.

File: evm-contracts/test/v0.6/FluxAggregator.test.ts (hunk 3 above)
Comment (se3000) [reply evm-contracts/test/v0.6/FluxAggregator.test.ts:1105]: Shoutout to @kaleofduty.

**Pull request merged**